### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by TomashKhamlai** [magento/magespecialist_ReCaptcha#6]
1) When 'Invisible reCaptcha' is set for Frontend, what type is used by Backend?
2) Is it possible to use 'reCaptcha v2' for Admin and 'Invisible reCaptcha' for the frontend? How to do this? 

I saw that there are only 2 fields for credentials, but I believe that there should be 4 fields because Google generates different keys for each variant of a check. If 2 fields are enough then how Magento administrator will know that he has to update the keys? People often want to play with settings especially this is the first time when they use Magento.

**Comment by phoenix128** [comment]
1. Backend only uses traditional reCaptcha. Invisible reCaptcha is a frontend only solution to avoid increasing checkout bouncing rate.
2. Just use invisible reCaptcha for frontend, backend is always the standard one.
3. Should we disable the Magento2 standard captcha?

**Comment by vkublytskyi** [comment]
I think question 2 is about fact that Google provides keys for concrete reCaptcha type. If admin want to configure V2 reCaptahca for admin and invisible reCaptahca for storefront it may confuse him that invisible reCaptcha should be generated on Google side as this is not obvious. Also, however Google allow to specify list of domains associated with reCaptcha site/secret keys pair it may be good to have possibility to specify these credentials per website.
To summarize we may consider possibility to have sperate config options for site/secret keys for backend and storefront. But for me it is more improvement and may be done in further releases

**Comment by TomashKhamlai** [comment]
This is example of showing/hiding additional fields in payment methods. Can be used for Invisible ReCaptcha keys too:
![peek 2018-02-21 17-08](https://user-images.githubusercontent.com/11827230/36487950-da256128-172a-11e8-9bd3-e9fbf3fd57d0.gif)

**Comment by phoenix128** [comment]
@TomashKhamlai , I'm not sure I understood what you mean. We are not talking about hiding fields, but about specifying different keys for backend.

@vkublytskyi , you already can specify different keys per different domains:

The following XML paths are already defined at storeview scope:
- `msp_securitysuite_recaptcha/general/private_key`
- `msp_securitysuite_recaptcha/general/public_key`

**Comment by vkublytskyi** [comment]
To sum up: Current implementation allows to specify different keys for all aspects. If admin and storefront uses different types of recaptcha then this may be resolved by specifying keys for backend as for "Default Config" and storefront keys for concrete website/store view. Any changes are up to module authors and not required for initial release as M2 bundled extension.
